src/validators/stop_times.rs

```diff
@@ -72,6 +73,31 @@ fn check_valid_stop_sequence(gtfs: &gtfs_structures::Gtfs) -> impl Iterator<Item
     })
 }
 
+fn check_times_increase(gtfs: &gtfs_structures::Gtfs) -> impl Iterator<Item = Issue> + '_ {
+    gtfs.trips
+        .values()
+        .map(|trip| {
+            trip.stop_times.iter().filter_map(move |st| {
+                match (st.arrival_time, st.departure_time) {
```

Comment:
Wondering if this is safe to compare with this https://github.com/rust-transit/gtfs-structure/issues/122#issue-1172599876 in mind?

Reply:
`std::time::Duration` [also implements][1] `Ord`, so I believe if the type of `arrival_time` & `departure_time` were changed this comparison would still work the same way (and if not the test will fail).

[1]: https://doc.rust-lang.org/std/time/struct.Duration.html#impl-Ord-for-Duration